granadilla/views.py

```diff
@@ -211,6 +211,34 @@ def get_context_data(self, **kwargs):
 group = login_required(GroupView.as_view())
 
 
+class ChangePasswordView(FormView):
+    """
+    function to change the user's password
+    """
+    model = models.LdapUser
+
+    template_name = 'granadilla/user_password.html'
+    form_class = LdapUserPassForm
+    success_url = 'confirmation/'
+
+    def get_context_data(self, **kwargs):
+        context = super(ChangePasswordView, self).get_context_data(**kwargs)
+        context['form_user'] = self.get_form()
+        return context
+
+    def form_valid(self, form):
+        form.save()
```

Comment:
You could use [Django's message framework](https://docs.djangoproject.com/en/1.11/ref/contrib/messages/#using-messages-in-views-and-templates) to display the confirmation message to the user; thus skipping the "confirmation" page.